specification/vmwarevirtustream/resource-manager/Microsoft.VMwareVirtustream/preview/2016-09-01-preview/vmwarevirtustream.json

```diff
@@ -0,0 +1,1304 @@
+            "Cancelled",
+            "Pending",
+            "Building",
+            "Updating"
+          ]
+        },
+        "circuit": {
+          "$ref": "#/definitions/Circuit"
+        },
+        "cluster": {
+          "$ref": "#/definitions/Cluster"
+        },
+        "clusters": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "endpoints": {
+          "$ref": "#/definitions/Endpoints"
+        },
+        "internetEnabled": {
+          "type": "boolean"
+        },
+        "identitySources": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/IdentitySource"
+          }
+        },
+        "vpc": {
+          "$ref": "#/definitions/Vpc"
+        }
+      }
+    },
+    "Cluster": {
+      "properties": {
+        "clusterId": {
+          "type": "integer",
+          "example": 1
+        },
+        "provisioningState": {
+          "type": "string",
+          "enum": [
+            "Succeeded",
+            "Failed",
+            "Cancelled",
+            "Updating"
+          ]
+        },
+        "clusterSize": {
+          "type": "integer",
+          "example": 4
+        },
+        "hosts": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        }
+      }
+    },
+    "PrivateCloudRequest": {
+      "properties": {
+        "location": {
+          "type": "string",
+          "example": "West US"
+        },
+        "properties": {
+          "$ref": "#/definitions/PrivateCloud"
+        },
+        "tags": {
+          "type": "object"
+        }
+      }
+    },
+    "PrivateCloudResponse": {
+      "properties": {
+        "id": {
+          "type": "string",
+          "readOnly": true
+        },
+        "location": {
+          "type": "string",
+          "example": "West US"
+        },
+        "name": {
+          "type": "string",
+          "readOnly": true
+        },
+        "properties": {
+          "$ref": "#/definitions/PrivateCloud"
+        },
+        "type": {
+          "type": "string",
+          "example": "Microsoft.VMwareVirtustream/privateClouds",
+          "readOnly": true
+        },
+        "tags": {
+          "type": "object"
+        }
+      },
+      "x-ms-azure-resource": true
+    },
+    "PrivateCloudResponseList": {
+      "properties": {
+        "value": {
+          "readOnly": true,
+          "items": {
+            "$ref": "#/definitions/PrivateCloudResponse"
+          }
+        }
+      }
+    },
+    "ClusterRequest": {
+      "properties": {
+        "location": {
+          "type": "string",
+          "example": "West US"
+        },
+        "properties": {
+          "$ref": "#/definitions/ClusterDetail"
+        }
+      }
+    },
+    "ClusterResponse": {
+      "properties": {
+        "id": {
+          "type": "string",
+          "readOnly": true
+        },
+        "location": {
```

Comment:
Will cluster have a location if it's a ProxyOnly resource?

Reply:
I'm going to remove it from  cluster here. The location is on private cloud.